kie-dmn-feel/pom.xml

```diff
@@ -71,6 +71,26 @@
 
   <build>
     <plugins>
+    
+        <plugin>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <configuration>
+            <fork>true</fork>
+            <meminitial>128m</meminitial>
+            <maxmem>512m</maxmem>
+            <showDeprecation>true</showDeprecation>
+            <showWarnings>true</showWarnings>
+            <source>1.8</source>
+            <target>1.8</target>
+            <testSource>1.8</testSource>
+            <testTarget>1.8</testTarget>
```

Comment:
I am not sure if we need/want it as top-level configuration. In any case, it should be enough to just include the particular configs you need to add/override. The source/target etc should be brought in from kie-parent.